src/Prelude.agda

```diff
@@ -81,10 +79,34 @@ import Data.Rational.Literals as ℚ
 instance Number-ℕ = ℕ.number
 instance Number-ℚ = ℚ.number
 
--- (pseudo)equality (for Maybe)
+-- (Pseudo)equality (for Maybe)
+open import Data.Maybe.Relation.Binary.Connected
+  using (Connected; connected?)
+
 _~_ : {A : Type} → Maybe A → Maybe A → Type
 _~_ = Connected _≡_
 
 instance
   ~? : {A : Type} {x y : Maybe A} → ⦃ DecEq A ⦄ → (x ~ y) ⁇
   ~? {A} {x} {y} ⦃ deqEq ⦄ = ⁇ (connected? (DecEq._≟_ deqEq) x y)
+
+-- Positive and negative part of integers
+open import Data.Integer using (sign; ∣_∣; _⊖_)
+open import Data.Integer.Properties using ([1+m]⊖[1+n]≡m⊖n)
+open import Data.Sign using (Sign)
```

Comment:
I prefer to have all imports at the top of the file.  It's more consistent with the rest of the code base, though there are exceptions to this rule of thumb.

Reply:
I decided to leave at the top the modules that are reexported, while leave those only needed to define things close to those things.